### DISCUSSION TITLE
Guideline needed

### DISCUSSION THREAD

**Discussion by smackdun** [php-edifact/edifact#112] [Q&A]
Dear team,

Thanks for your hard work.

I'm not a dev so could you please guide me more specific how to use this tool?

Best regards,

**Comment by sabas** [comment]
What's your expected usage?

**Reply by smackdun** [reply]
Hi sabas,

Thank for your reply.

What I expect is that i can use this as a parser where i can input an .edi file and it will show values of each segment, sub-segment and elements.

Can this be achived?

Best regards,
Hai

**Reply by sabas** [reply]
Hi, this is a library which gives you some tools to process a file. If you are not a developer I can suggest you may use some ready-made software?
For example http://www.edifactviewer.com/
